packages/genesys-spark-components/src/components/beta/gux-tooltip-beta/gux-tooltip.tsx

```diff
@@ -100,7 +110,7 @@ export class GuxTooltip {
 
   render(): JSX.Element {
     return (
-      <Host id={this.id} role="tooltip">
+      <Host id={this.id} role={this.tooltipShown ? 'tooltip' : null}>
```

Comment:
are you sure we have to put the role on this element? Would it work if the role was on the base element?